typescript/cli/src/utils/fork.ts

```diff
@@ -128,3 +129,18 @@ export const getLocalProvider = (
 
   return new providers.JsonRpcProvider(url);
 };
+
+/**
+ * Retrieves the balance of an account.
+ * @param address the address to retrieve the balance of
+ * @returns the balance of the account
+ */
+export const getAccountBalance = async (address: Address) => {
```

Comment:
If it's anvil specific let's rename it for clarity please

Reply:
This is a +1 for class encapsulation– what are you thoughts on getting that kicked off by wrapping this fork util in a class?

Something like `class ForkUtil` (may have constructor params– perhaps port #s for example, since we will expand to multi-network forking)

instantiated like `const forkUtil = new ForkUtil();`

used like `const impersonatedAccountBalance = forkUtil.getAccountBalance()`

Reply:
I'm weakly against Class encapsulation in cases like this because they're so redundant with the encapsulation that JS modules already provide. Every file gets private state (via any top-level vars) and an explicit interface (via the file's imports/exports), so wrapping things in a class Java-style doesn't actually get you much. 

That said, classes makes sense where you intend to actually use OO patterns such as inheritance, which is why we use them in the SDK's deployers/checkers/etc. 

So yeah I wouldn't spend any time wrapping up all our various util fns in classes but if you think we'll want a strict hierarchy then I'm open to it in this case.

Reply:
Makes complete sense– generally speaking, I bring up class encapsulation as way for us to run context-driven CLI commands in the future (deduping code between commands, like warp and core), which would step closer to OO like the sdk

I am weakly for that beginning here, and as you mentioned it really is not necessary for context-driven commands at any point, so I will leave fork.ts as is and refactor to getAnvilAccountBalance.

Would love to continue chatting offline about the command flow at a high-level 👍